docs/visual-basic/language-reference/functions/ctype-function.md

```diff
@@ -46,6 +46,16 @@ If a conversion fails at run time, the appropriate exception is thrown. If a nar
 
 If the data type of `expression` or `typename` is a class or structure you've defined, you can define `CType` on that class or structure as a conversion operator. This makes `CType` act as an *overloaded operator*. If you do this, you can control the behavior of conversions to and from your class or structure, including the exceptions that can be thrown.
 
+## Conversion Keywords  
```

Comment:
```suggestion
## Conversion keywords
```

Reply:
Hi @Youssef1313,

Thanks for reviewing.

Looking back through each link (e.g. TryCast, DirectCast) in the Conversion Keywords table on each of those pages, the section header has the capitalization as shown in my commit.  I'd be inclined to keep them all the same.